src/addon/info.hpp

```diff
@@ -87,6 +87,27 @@ struct addon_info
 		this->read(cfg);
 	}
 
+	addon_info(const addon_info& o)
+		: id(o.id)
+		, title(o.title)
+		, description(o.description)
+		, icon(o.icon)
+		, version(o.version)
+		, author(o.author)
+		, size(o.size)
+		, downloads(o.downloads)
+		, uploads(o.uploads)
+		, type(o.type)
+		, tags(o.tags)
+		, locales(o.locales)
+		, core(o.core)
+		, depends(o.depends)
+		, feedback_url(o.feedback_url)
+		, updated(o.updated)
+		, created(o.created)
+		, local_only(o.local_only)
+	{}
+
```

Comment:
why not `= default` here instead

Reply:
I didn't realize that was a thing I could do.  I'll change that.